cmd/ddev/cmd/debug-router-nginx-config.go

```diff
@@ -47,7 +48,7 @@ var DebugRouterNginxConfigCmd = &cobra.Command{
 }
 
 func init() {
-	if !globalconfig.DdevGlobalConfig.UseTraefik {
+	if !globalconfig.IsTraefikRouter() {
```

Comment:
That's dangerous, I'd create a second func IsNginxRouter instead.

Reply:
Changed to direct comparison; this is a very obscure location though, in an obsolete usage, not worth too much worry.